rolepermissions/management/commands/sync_roles.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import Group
+from django.contrib.auth import get_user_model
+from rolepermissions import roles
+
+
+class Command(BaseCommand):
+    ROLEPERMISSIONS_MODULE = getattr(settings, 'ROLEPERMISSIONS_MODULE', 'roles.py')
+    help = "Synchronize auth Groups and Permissions with UserRoles defined in %s."%ROLEPERMISSIONS_MODULE
+    version = "1.0.0"
+
+    def get_version(self):
+        return self.version
+
+    def add_arguments(self, parser):
+        # Optional argument
+        parser.add_argument(
+            '--reset_user_permissions',
+            action='store_true',
+            dest='reset_user_permissions',
+            default=False,
+            help='Re-assign all User roles -- resets user Permissions to defaults defined by role(s) !! CAUTION !!',
+        )
+
+    def handle(self, *args, **options):
+        # Sync auth.Group with current registered roles (leaving existing groups intact!)
+        for role in roles.RolesManager.get_roles() :
+            group, created = Group.objects.get_or_create(name=role.get_name())
```

Comment:
Let's create a method in `AbstractUserRole` to deal with the creation of `Group`s. We can call it here and [here](https://github.com/vintasoftware/django-role-permissions/blob/master/rolepermissions/roles.py#L60)

Reply:
Good idea. Will do.